manifests/peer.pp

```diff
@@ -37,11 +37,21 @@
 define gluster::peer (
   $pool = 'default',
   $fqdn = $facts['networking']['fqdn'],
+  Optional[Boolean] $force_binary = false,
```

Comment:
when it's false, you can't pass undef. So Optional doesn't make any sense. Can we just use this?
```suggestion
  Boolean $force_binary = false,
```